server/test/controllers/applicant/ApplicantRoutesTest.java

```diff
@@ -40,6 +40,10 @@ private Counts getApplicantIdInProfileCounts() {
     for (MetricFamilySamples mfs : Collections.list(registry.metricFamilySamples())) {
       if (mfs.name.equals("applicant_id_in_profile")) {
         for (MetricFamilySamples.Sample sample : mfs.samples) {
+          if (!sample.name.equals("applicant_id_in_profile_total")) {
```

Comment:
Prior version of the Prometheus library only had `applicant_id_in_profile_total`. With the new version there is an additionally generated option named `applicant_id_in_profile_created`. The tests were written assuming only the `applicant_id_in_profile_total` was there.